api/controllers/auth/controller.go

```diff
@@ -16,32 +23,48 @@ type Controller interface {
 var _ Controller = (*AuthController)(nil)
 
 type AuthController struct {
-	password string
+	passwordHash []byte
 }
 
-type AuthControllerOption func(*AuthController)
-
-func NewAuthController(password string, opts ...AuthControllerOption) (*AuthController, error) {
-	controller := &AuthController{
-		password: password,
+// NewAuthController creates a new auth controller from a plaintext password
```

Comment:
i think there should only be a single auth controller method that accepts a hash, and update `runManagerExperienceInstall` in the cli code to generate the hash from the provided password and pass it to the api.